components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -68,14 +63,11 @@ public class LegacyJobOrchestrator : IJobOrchestrator, IPublisherConfigServices,
             _publishedNodesEntrys = new List<PublishedNodesEntryModel>();
 
             RefreshJobFromFile();
-
-            var file = Path.GetFileName(_legacyCliModel.PublishedNodesFile);
-            _fileSystemWatcher = new FileSystemWatcher(directory, file);
-            _fileSystemWatcher.Changed += _fileSystemWatcher_Changed;
-            _fileSystemWatcher.Created += _fileSystemWatcher_Created;
-            _fileSystemWatcher.Renamed += _fileSystemWatcher_Renamed;
-            _fileSystemWatcher.Deleted += _fileSystemWatcher_Deleted;
-            _fileSystemWatcher.EnableRaisingEvents = true;
+            _publishedNodesProvider.FileSystemWatcher.Changed += _fileSystemWatcher_Changed;
```

Comment:
Logs of file system notifications should be handled by `PublishedNodesProvider` class itself.

Reply:
I think that the file change logs should stay in `LegacyJobOrchestrator` class since the client of `PublishedNodesProvider` class should decide what to do with the notifications and whether to act on them or not.